scripts/cibuild

```diff
@@ -23,8 +23,11 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
             -e "s/%%OPENJDK_VERSION%%/$OPENJDK_VERSION/" \
             "Dockerfile-$VARIANT.template" > Dockerfile
 
+        # Strip the patch number off of GDAL_VERSION for tagging
+        GDAL_MAJOR_VERSION="${GDAL_VERSION:0:3}"
```

Comment:
There may be a more centralized way of doing this. The first thing that came to mind was requiring that the major version be its own environment variable, but I didn't like the duplication there. Other ideas are welcome.

Reply:
This works, but could break down in the event that the MINOR version is double digits. Might be able to be more precise with `cut`:

```bash
echo "${GDAL_VERSION}" | cut -d"." -f1,2
```

Reply:
Much better. Slightly ashamed that I decided to go with a spotty string manipulation after [getting burned by a spotty string manipulation yesterday](https://github.com/raster-foundry/load-testing/pull/2).